Tests/OpenAPIRuntimeTests/Conversion/Test_Configuration.swift

```diff
@@ -12,7 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 import XCTest
-@_spi(Generated) import OpenAPIRuntime
+@_spi(Generated) @testable import OpenAPIRuntime
```

Comment:
OOI, why was the `@testable` import needed here? Could we write the tests to use only the API?

Reply:
Without it, I can't access the `encoder` directly to try to encode a few sample values. Otherwise I'd have to go through the SPI for generated code, and those require creating a bit more state to invoke.

It's intentional, we _want_ to hide `Foundation.JSONEncoder` from the API (and SPI), but it does require us to either use `@testable` here or make the test go through a bit more unrelated code.

Reply:
I should have been clearer. I could see what was using the `@testable` import, I was asking why we didn't exercise this new functionality through the public API only, which I've been leaning to these days because it prevents accidentally releasing things that pass tests but haven't been exposed properly in the API.

For clarity this is not a blocking comment and I'm happy to stick with the way it is.

Reply:
No you bring up a good point, let me update and drop the `@testable` import.

Reply:
Ok updated.